Moonscraper Chart Editor/Assets/Scripts/Game/UI/Inspectors/StarpowerPropertiesPanelController.cs

```diff
@@ -75,7 +78,7 @@ void UpdateTogglesDisplay()
             Debug.LogError("No starpower loaded into note inspector");
         }
 
-        drumFillToggle.isOn = flags.HasFlag(Starpower.Flags.ProDrums_Activation);
+        drumFillToggle.isOn = drumsMode ? flags.HasFlag(Starpower.Flags.ProDrums_Activation) : false;
```

Comment:
`drumFillToggle.isOn = drumsMode && flags.HasFlag(Starpower.Flags.ProDrums_Activation);`

Reply:
It's a more concise way to write what I wrote?